README.md

```diff
@@ -2,20 +2,16 @@
 
 #### For Users
 
-* <a href="https://powerofus.force.com" target="_blank">Ask questions or get help</a>
-* <a href="https://github.com/SalesforceFoundation/NPSP/issues/new" target="_blank">Log a confirmed Issue</a> or <a href="https://powerofus.force.com/hub-ideas" target="_blank">Feature Request</a>
+* <a href="https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHitSAE?tab=discussion&sort=LAST_MODIFIED_DATE_DESC" target="_blank">Ask questions or get help</a>
+* <a href="https://github.com/SalesforceFoundation/NPSP/issues/new" target="_blank">Log a confirmed Issue</a> or <a href="https://ideas.salesforce.com/s/search#t=All&sort=relevancy" target="_blank">Feature Request</a>
 * <a href="https://powerofus.force.com/NPSP_Documentation" target="_blank">User Documentation</a>
-* Check out existing <a href="https://github.com/SalesforceFoundation/NPSP/labels/bug" target="_blank">bugs</a> and <a href="https://powerofus.force.com/hub-ideas?product=NPSP" target="_blank">feature and enhancement requests.</a>
+* Check out existing <a href="https://github.com/SalesforceFoundation/NPSP/labels/bug" target="_blank">bugs</a> and <a href="https://ideas.salesforce.com/s/search#t=All&sort=relevancy&f:@sfcategoryfull=[Nonprofit%7CNonprofit%20Success%20Pack]" target="_blank">feature and enhancement requests.</a>
```

Comment:
@sam-knox  - and same with this mention of 'bugs' -- maybe link to Known Issues instead?
https://trailblazer.salesforce.com/issues_index?tag=SFDO%20Nonprofit%20Success%20Pack

Reply:
@jesslopez-sf : Yeah we can update that. The switch to Known Issues happened (or we knew of it) well after we made these initial updates to remove the Hub references. I can ask the devs about this, but being that it's near the end of the development cycle I'll wait until after feature freeze.

I imagine there's similar updates needed in other repos too.